qa/workunits/suites/fsstress.sh

```diff
@@ -8,7 +8,7 @@ wget -q -O ltp-full.tgz http://download.ceph.com/qa/ltp-full-20091231.tgz
 tar xzf ltp-full.tgz
 pushd ltp-full-20091231/testcases/kernel/fs/fsstress
 make
-BIN=$(realpath fsstress)
+BIN=$(readlink -f fsstress)
```

Comment:
This LGTM but I dunno if it's anymore common than realpath. Apparently so if it solves your problem.

Reply:
I think it’s more about the fact that realpath is in its own package whereas readlink is part of coreutils.